Composer/packages/client/src/components/AddRemoteSkillModal/CreateSkillModal.tsx

```diff
@@ -0,0 +1,279 @@
+              {formatMessage(' learn how to build a skill ')}
+            </Link>
+            {addSkillDialog.SKILL_MANIFEST_FORM.afterSubText}
+          </div>
+          <Separator />
+          <Stack horizontal horizontalAlign="start" styles={{ root: { height: 300 } }}>
+            <div style={{ width: '50%' }}>
+              <TextField
+                required
+                errorMessage={formDataErrors.manifestUrl}
+                label={formatMessage('Skill Manifest Url')}
+                value={formData.manifestUrl || ''}
+                onChange={handleManifestUrlChange}
+              />
+              {skillManifest?.endpoints?.length > 1 && (
+                <Dropdown
+                  label={formatMessage('Endpoints')}
+                  options={options}
+                  defaultSelectedKey={skillManifest.endpoints[0].msAppId}
+                  responsiveMode={ResponsiveMode.large}
+                  onChange={(e, option?: IDropdownOption) => {
+                    if (option) {
+                      setFormData({
+                        ...formData,
+                        endpointName: option.text,
+                      });
+                    }
+                  }}
+                />
+              )}
+            </div>
+            {showDetail && (
+              <Fragment>
+                <Separator vertical styles={{ root: { padding: '0px 20px' } }} />
+                <div style={{ minWidth: '50%' }}>
+                  {skillManifest ? <SkillDetail manifest={skillManifest} /> : <LoadingSpinner />}
+                </div>
+              </Fragment>
+            )}
+          </Stack>
+          <Stack>
+            <Separator />
+            <StackItem align={'end'}>
+              <DefaultButton data-testid="SkillFormCancel" text={formatMessage('Cancel')} onClick={onDismiss} />
+              {skillManifest ? (
+                isUsingAdaptiveRuntime(runtime) && luFiles.length > 0 ? (
+                  <PrimaryButton
+                    styles={{ root: { marginLeft: '8px' } }}
+                    text={formatMessage('Next')}
+                    onClick={(event) => {
+                      setTitle(selectIntentDialog.SELECT_INTENT(dialogId, skillManifest.name));
+                      setShowIntentSelectDialog(true);
+                    }}
+                  />
+                ) : (
+                  <PrimaryButton
+                    styles={{ root: { marginLeft: '8px' } }}
+                    text={formatMessage('Done')}
+                    onClick={(event) => {
+                      addRemoteSkill(formData.manifestUrl, formData.endpointName);
+                    }}
+                  />
+                )
+              ) : (
+                <PrimaryButton
+                  disabled={!formData.manifestUrl || formDataErrors.manifestUrl !== undefined}
+                  styles={{ root: { marginLeft: '8px' } }}
```

Comment:
This style is used 4 times. maybe just create an object at the top and resuse it inside the PrimaryButton